docs/elasticsearch-spec.asciidoc

```diff
@@ -359,64 +359,51 @@ The init container inherits the image of the main container image if one is not
 [id="{p}-update-strategy"]
 === Update strategy
 
-The Elasticsearch cluster configuration can be updated at any time to:
+Users can choose to constrain the operator on the resources it can use when updating using `updateStrategy' like below.
```

Comment:
How about:

> You can use `updateStrategy' to constrain the operator on the resources it can use.

Reply:
Ah, sorry, I forgot to push one change from https://github.com/elastic/cloud-on-k8s/pull/1883#discussion_r332007714, it was changing the first paragraph to the following:

```
Sometimes the operator must take a Pod down, for example, to restart a node and apply a new configuration value. Other times the operator must spin up pods above what is currently in the spec, say to migrate to a new node set. You can limit the number of these simultaneous changes  using an `updateStrategy` as follows:
```

Should we go with the below anyway?
```
You can use `updateStrategy' to constrain the operator on the resources it can use.
```

Reply:
If it doesn't change the meaning of the overall sentence, then yes.